e2e/pages/toolbar.ts

```diff
@@ -19,6 +20,20 @@ module.exports = {
         .waitForElementVisible(element)
         .click(element);
     },
+    changeLocale(index: number) {
+      const img = `.flag-container>:nth-child(${index + 1})`;
+      return (
+        this.clickNavElement('@localePicker')
+          .waitForElementVisible(img)
+          .click(img)
+          // We need to click outside of the navigation drawer to dismiss it, so
+          // just use an arbitrary large offset from the menu icon in the upper-left
+          // corner of the window.
+          .moveToElement('@menuIcon', 500, 0, () => {
```

Comment:
Is it possible to use "hide-overlay" instead of simulating a click outside?

Reply:
(as discussed in-person)

i think that hide-overlay just controls whether the rest of the screen is dimmed while the nav drawer is open. the larger problem is that hide-overlay (and more importantly, v-navigation-drawer's v-model attribute) are vue attributes, while this is an e2e test that doesn't have access to any of that -- it can just find and click on different elements.

another approach i tried but didn't mention in the comment was to close the drawer using the keyboard, but it strangely doesn't respond to the escape key by default and i didn't want to mess around with adding a key event handler.